tests/unit/test_vehicle.py

```diff
@@ -0,0 +1,8 @@
+from tests.conftest import access
+
+
+def test_vehicle_constructor_with_flags(chevy_volt_with_flags):
```

Comment:
Rather than using a fixture could we just create the vehicle in this test itself? 

rationale: this vehicle is very specific to this test case so doesn't make sense for it to be a fixture
rationale #2: since this is a unit test we don't need to get a real vehicle id so we can just directly call `sc.Vehicle` rather than having to get vehicles using an access token